File: Microservices/BillingService/BillingService.Domain/Models/ClaimOptions.cs
using BillingService.Domain.DataObjects.Base;
using System.Collections.Generic;

namespace BillingService.Domain.Models
{
    public class ClaimOptions
    {
        public List<BasicOption> Clients { get; set; }
        public List<BasicOption> Locations { get; set; }
        public List<BasicOption> Members { get; set; }
        public List<BasicOption> LocationCodes { get; set; }
        public List<int> ClaimIds { get; set; }
        public List<BasicOption> UnitTypes { get; set; }

        public List<BasicOption> RenderingProviders { get; set; }
        public List<BasicOption> ReferringProviders { get; set; }
        public List<BasicOption> BillingProviders { get; set; }
        public List<BasicOption> ServiceFacilities { get; set; }
    }
}

File: billing_ui_app/src/app/core/models/billing/claim-options.ts
import { BasicOption } from '@core/models/common';


export class ClaimOptions {
    public clients: BasicOption[] = [];
    public locations: BasicOption[] = [];
    public members: BasicOption[] = [];
    public locationCodes: BasicOption[] = [];
    public claimIds: number[] = [];
    public unitTypes: BasicOption[] = [];
    public renderingProviders: BasicOption[] = [];
    public referringProviders: BasicOption[] = [];
    public serviceFacilities: BasicOption[] = [];
    public billingProviders: BasicOption[] = [];
}